src/ApiKey/ApiKey.php

```diff
@@ -4,8 +4,20 @@
 
 namespace CultuurNet\UDB3\ApiGuard\ApiKey;
 
-use ValueObjects\StringLiteral\StringLiteral;
-
-final class ApiKey extends StringLiteral
+final class ApiKey
 {
+    /**
+     * @var string
+     */
+    private $value;
+
+    public function __construct(string $value)
+    {
+        $this->value = $value;
+    }
+
+    public function getValue(): string
```

Comment:
I wonder if this should be a `toString()` for consistency although the value is a string? Because other value objects with other types then `string` will have a `toString` method.

Reply:
Good point. I don't have any strong preference. @bertramakers ?

Reply:
I don't have any numbers right now but I feel like we have more `toString()` methods than `getValue()` methods on custom value objects in other (udb3) repo's right now, so I'd also lean more towards `toString()`. But in the end as long as it's all decoupled I don't have a strong preference, it's normal that some common methods like that will start to deviate in naming etc. It's a normal side effect of not having everything extend from a common parent class of interface I think, and one that I'm okay to live with if it means not having a shared dependency in everything 😄